buildSrc/src/main/groovy/GwtTools.groovy

```diff
@@ -83,8 +82,14 @@ class GwtTools {
     static void applyDefaults(Project p, GwtExtension gwt, boolean compile = false) {
         gwt.gwtVersion = Classpaths.GWT_VERSION
         gwt.jettyVersion = Classpaths.JETTY_VERSION
+        p.configurations.all { Configuration c ->
```

Comment:
As part of migrating GWT away from Google and letting the open source community manage releases, we had to change the groupId of releases - effectively "what domain owns this" so that maven central can validate ownership.

Since none of the community members owns access to `google.com`, the owners of `gwtproject.org` instead are releasing artifacts now - the final release using `com.google.gwt` was 2.9, the 2.10 release on `com.google.gwt` is just metadata saying "go look over at `org.gwtproject` instead", and the 2.11+ releases will only be on `org.gwtproject`. This build configuration is necessary to tell the gradle plugin how to get copies of GWT that matches our expected versions, since there will never be a `com.google.gwt` release of version 2.11.0 (even just with metadata saying "go check the new groupId").

Reply:
That's the kind of detail I'd like in the PR description :)